pymc_marketing/mmm/base.py

```diff
@@ -363,6 +363,7 @@ def plot_posterior_predictive(
         self,
         original_scale: bool = False,
         add_hdi: bool = True,
+        hdi_list: list[float] | None = None,
```

Comment:
Shall we add a warning when we have `add_hdi=False` and we have a non None `hdi_list`? (In this case we ignore the hdi params?)

Can we please add one simple test case in https://github.com/pymc-labs/pymc-marketing/blob/main/tests/mmm/test_plotting.py#L195 where we pass the hdi list?

Reply:
Is there a way to do this with one parameter? That would be ideal

Reply:
Just addressed @juanitorduz 's requests, but agree that having both kwargs is excessive. Maybe we can just remove `add_hdi` and if the user doesn't want one then they provide a `None` or `[]` to `hdi_list`?

Reply:
Personally I think we definitely want HDI's by default. 

Though I'd make the default `[0.94]` to align with `arviz`. A user could then change to a list or `None` or `[]`?

Reply:
Ok. That last commit (https://github.com/pymc-labs/pymc-marketing/pull/1096/commits/38415454e70394dd712121c67f7382cc08bb82f8) now leaves us with:

* default is to plot HDI with 0.94
* `hdi_list=None` ends up with the default. Because the pre-commit checks won't let me set the default value to be `[0.94]` then it has to be `None` and then set that to the default value in the function body.
* So if you want to omit plotting the HDI, then you have to do that by `hdi_list=[]`.

Reply:
If this is acceptable, I should maybe rebuild the docs so that the posterior prediction plots show the update... only the 94% HDI.